src/query-builder.ts

```diff
@@ -36,8 +36,8 @@ export function fql(
  * function
  */
 export class Query {
-  readonly #queryFragments: ReadonlyArray<string>;
-  readonly #queryArgs: (QueryValue | Query)[];
+  readonly queryFragments: ReadonlyArray<string>;
+  readonly queryArgs: (QueryValue | Query)[];
```

Comment:
Seems pretty leaky to expose this to users. How about we pass these as arguments to TaggedTypeFormat.encodeInterpolation so these can stay private.